packages/neutrino-preset-web/index.js

```diff
@@ -67,14 +67,13 @@ module.exports = (neutrino) => {
   neutrino.use(htmlTemplate, neutrino.options.html);
   neutrino.use(namedModules);
   neutrino.use(compileLoader, {
-    include: [neutrino.options.source, neutrino.options.tests],
+    include: [neutrino.options.source, neutrino.options.tests, require.resolve('./polyfills.js')],
```

Comment:
If you mean to avoid './polyfills.js' it will not work. `babel-preset-env` just want to see some where the importing of 'babel-polyfill'. And it doesn't see it if this file is an entry point itself.